specs/commandline/policy.md

```diff
@@ -0,0 +1,696 @@
+# notation policy
+
+## Description
+
+Use `notation policy` command to add/update/show/delete trust policies. As part of signature verification workflow, user needs to configure the trust policies to specify trusted identities that sign the artifacts, and the level of signature verification to use. For more details, see [trust policy spec](https://github.com/notaryproject/notaryproject/blob/main/specs/trust-store-trust-policy.md#trust-policy).
+
+An example of `trustpolicy.json`:
+
+```json
+{
+    "version": "1.0",                                                   // version info
+    "trustPolicies": [                                                  // list of trust policy statements
+        {
+            "name": "wabbit-networks-dev",                              // Name of the first policy statement
+            "registryScopes": [ "dev.wabbitnetworks.io/net-monitor" ],  // The registry artifacts to which the policy applies
+            "signatureVerification": {                                  // The level of verification - strict, permissive, audit, skip
+                "level": "strict"
+                "override" : {
+                     "expiry" : "log",
+                     "authenticity": "log"
+                }
+            },
+            "trustStores": [ "ca:wabbit-networks-dev" ],                // The trust stores that contains the X.509 certificates
+            "trustedIdentities": [                                      // Identities that are trusted to sign the artifact.
+                "x509.subject: C=US, ST=WA, L=Seattle, O=Example, OU=Dev, CN=wabbit-networks.io"
+            ]
+        },
+        {
+            "name": "wabbit-networks-prod",                             // Name of the second policy statement.
+            "registryScopes": [ "prod.wabbitnetworks.io/net-monitor" ],       
+            "signatureVerification": {                                
+                "level": "permissive"
+            },
+            "trustStores": [ "ca:wabbit-networks-prod" ],                  
+            "trustedIdentities": [                                    
+                "x509.subject: C=US, ST=WA, L=Seattle, O=Example, OU=Prod, CN=wabbit-networks.io"
+            ]
+        }
+    ]
+}
+```
+
+## Outline
+
+### notation policy command
+
+```text
+Manage trust policies for signature verification.
+
+Usage:
+  notation policy [command]
+
+Available Commands:
+  add           Add trust policies
+  delete        Delete trust policies
+  list          List trust policies
+  update        Update trust policies
+
+Flags:
+  -h, --help   help for policy
+```
+
+### notation policy add
+
+```text
+Add trust policies.
+
+Usage:
+  notation policy add (--file <policy_file_path> | --name <policy_name>) [flags]
+
+Flags:
+      --custom-level       stringArray   {key}={value} pairs that represent a custom level to override existing verification level
```

Comment:
For all the fields that have type `stringArray` can we use `string`? customer can use them multiple times to pass multiple values. Also, in this case default value for --scope becomes `*`